development/ProjectOntology.owl

```diff
@@ -289,19 +210,6 @@
     
 
 
-    <!-- https://w3id.org/enigma#hasProjectLead -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasProjectLead">
```

Comment:
Why is this one removed? Are we using PI instead?

Reply:
We are using hasPersonRole_ProjectLead

Reply:
ok